ecc/enclave_chaincode_test.go

```diff
@@ -118,7 +108,8 @@ func TestEnclaveChaincode_Invoke_Auction(t *testing.T) {
 
 	th.CheckInvoke(t, stub, create_args)
 
-	executor := executor.NewConcurrent("Client", 16)
+	// note that shim.MockStub is not thread-safe! Therefore we better use a single executor here
```

Comment:
In this context, interesting asplos'19 paper: https://blog.acolyer.org/2019/05/17/understanding-real-world-concurrency-bugs-in-go/ ...

Reply:
nice morning read